### ISSUE TITLE
Implement ZBX_LISTENPORT for zabbix-agent

### ISSUE THREAD

**Issue by verwilst** [zabbix/zabbix-docker#402]
Currently you can specify the listenip, but not the listenport.
This comes in handy when you want to listen on a different port when network_mode = host.

**Comment by dengruquan** [comment]
you can add the parameter at your docker-entrypoint.sh
    update_config_var $ZBX_AGENT_CONFIG "ListenPort" "${ZBX_LISTENPORT}"
    if [ -n "${ZBX_LOGTYPE}" ]; then
        update_config_var $ZBX_AGENT_CONFIG "LogType" "${ZBX_LOGTYPE}"
        update_config_var $ZBX_AGENT_CONFIG "LogFile" "${ZBX_LOGFILE}"
    fi

[root@kg zabbix_agent]# cat docker-compose.yml 
version: "3"
services:
    zabbix_agent:
        image: zabbix/zabbix-agent:alpine-3.0-latest
        container_name: zabbix_agent
        ports:
          - "3050:3050"
        env_file:
          - ./etc/env_agent
        volumes:
          - /etc/localtime:/etc/localtime:ro
          - ./etc/docker-entrypoint.sh:/usr/bin/docker-entrypoint.sh:ro      
          - ./etc/zabbix_agentd.d:/etc/zabbix/zabbix_agentd.d    
          - ./scripts:/data/zabbix/scripts
          - ./logs:/tmp
        #network_mode: host
        networks:
          - zabbix
networks:
    zabbix:
        external: true

the first step: you can use docker cp to copy the docker-entrypoint.sh from zabbix_agent, 
second step: add the parameter you like at the docker-entrypoint.sh
third step: volume it on you docker-compose.yml

**Comment by verwilst** [comment]
Yeah, I can add it manually and recreate the image, but the point of this issue to have it ready "out-of-the-box" :-)

**Comment by dengruquan** [comment]
Well, I like this way: I can personalize it if I just change the script content instead of recreate the image.
